main.py

```diff
@@ -902,6 +986,33 @@ def handle_interface_link_up(self, interface, event):
             self._intfs_updated_at[interface.id] = event.timestamp
         self.handle_link_up(interface)
 
+    def _notify_link_from_interface(self, interface):
+        """Determines whether to send a notification."""
+        link = interface.link
+        if interface.id == link.endpoint_a.id:
+            other_intf = link.endpoint_b
+        else:
+            other_intf = link.endpoint_a
+        if other_intf.is_enabled() and not interface.is_enabled():
+            self.notify_link_enabled_state(link, "enabled")
+
+    @tenacity.retry(
+        reraise=True,
+        stop=stop_after_attempt(3),
+        wait=wait_combine(wait_fixed(3), wait_random(min=2, max=7)),
+        before_sleep=before_sleep,
+        retry=retry_if_exception_type(HTTPException),
```

Comment:
Notice that this is not going to retry for timeout. In other NApps, I've been retrying for `httpx.RequestError` since that's a parent of `TimeoutException` and  other transient exceptions that can happen during IO. 

But to also do it this way, you need to also reraise `httpx.RequestError`, check out [an example here](https://github.com/kytos-ng/telemetry_int/blob/master/kytos_api_helper.py#L111), this is the simplest way that I've found so far to handle this, let me know if you see something simpler or more convenient.

Reply:
Fixed